jetty-websocket/websocket-javax-client/src/main/java/org/eclipse/jetty/websocket/javax/client/internal/JavaxWebSocketClientContainer.java

```diff
@@ -95,6 +112,11 @@ public JavaxWebSocketClientContainer(WebSocketComponents components, Function<We
         this.frameHandlerFactory = new JavaxWebSocketClientFrameHandlerFactory(this);
     }
 
+    public void allowShutdownWithContextHandler(boolean allowShutdownWithContextHandler)
```

Comment:
This needs to be a proper setter method, starting with `set` and the getter should be there too, and used in the code.